cps/src/main/resources/org/jenkinsci/plugins/workflow/editor/workflow-editor.jelly

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:f="/lib/form">
+    <st:documentation>
+        <st:attribute name="script">
+            The script.
+        </st:attribute>
+    </st:documentation>
+    <j:choose>
+        <j:when test="${h.isUnitTest}">
+            <!-- HtmlUnit and ACE do not seem to mix. -->
```

Comment:
Yeah, it does a lot of async initialization, which means it's a PITA to integrate with HtmlUnit.